infrastructure/modules/app-cluster/variables.tf

```diff
@@ -202,3 +202,17 @@ variable "layers" {
   description = "A list of the layers that the rAPId instance will contain"
   default     = ["default"]
 }
+
+variable "custom_username_regex" {
+  type        = list(string)
+  description = "A list containing the regex expression for conditional user validation and a string to test against. The input must be a terraform list variable composed of two strings. The first string is the regex expression and the second string is the test string for said regular expression."
+  # Set default to a regex expression for basic api username validation
+  default = ["[a-zA-Z][a-zA-Z0-9@._-]{2,127}", "AUser1"]
```

Comment:
We want a default to be None and this to become a optional variable. Also doesn't make sense for this to be a list of strings as we just want this to be a regex string